app/components/goals-list/goals-list.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const goalsList = props => {
+
+  const goals = props.store.goals;
+
+  const goalsListItems = goals.map(goal => {
+    return <li key={ goal.id }>{ goal.name }</li>
+  });
+
+  const style = {
+    background: 'url(app/public/images/goals-list/background.png)',
```

Comment:
erm yeah, i only did this because i didn't know how to handle the images with sass/web pack 🙈. Going to look into it now...